src/axolotl/utils/data/rl.py

```diff
@@ -204,7 +204,12 @@ def load_split(dataset_cfgs, _cfg):
             else:
                 eval_dataset = load_split(cfg.test_datasets, cfg)
         if not eval_dataset:
-            eval_dataset = None
+            if cfg.val_set_size:
+                ds_w_test_split = train_dataset.train_test_split(
+                    test_size=cfg.val_set_size, seed=cfg.seed
+                )
```

Comment:
Do we need to calculate hash like it's done in SFT?

https://github.com/axolotl-ai-cloud/axolotl/blob/8b33ae1c4f5078e0b3d2cb31bb6856692aa3d993/src/axolotl/utils/data/sft.py#L419-L439